types/node/index.d.ts

```diff
@@ -2190,11 +2190,13 @@ declare module "child_process" {
         keepOpen?: boolean;
     }
 
+    export type StdioOptions = "pipe" | "ignore" | "inherit" | Array<("pipe" | "ipc" | "ignore" | stream.Stream | number)>;
```

Comment:
`null` and `undefined` are also allowed as types in the array

Reply:
Good point, I overlooked those in the docs.